### ISSUE TITLE
渲染出来的效果不够清晰

### ISSUE THREAD

**Issue by touwenameng** [PeizhiYan/gaussian-dejavu#4]
渲染出来的头像清晰度不足，而且牙齿也很模糊，是否有提高效果的办法

**Comment by PeizhiYan** [comment]
你好！我们目前的这个工作非常依赖比较准确的flame head tracking结果，如果tracking不是很准确效果会不好。并且因为没有考虑光照，如果视频里光照在脸上不均匀的话也会造成效果不好。
高斯UV map的大小选取也会影响效果，如果用大一些的UV map size，比如320x320会比用较小的尺寸渲染结果清晰。

**Comment by touwenameng** [comment]
感谢回复，另外有个问题是牙齿，目前版本牙齿不大清楚，也是UV map size的问题吗

**Comment by PeizhiYan** [comment]
牙齿问题我认为更有可能是我们用于牙齿部位的高斯数量太少。 有一个比较工程的办法就是做一个牙齿的模型然后缝合到口腔里，但这样做要麻烦些 。我们口腔的那部分是用了interpolation的方法生成的（见https://peizhiyan.github.io/docs/dejavu/wacv-2025-supp.pdf）这样做简单地把FLAME口腔补上了，但是并没有直接解决牙齿的问题。

**Comment by alasokolova** [comment]
Hey @PeizhiYan 
I faced the issue with blurry results, too.
If i try to train the model for larger UV map size, like 320, as you write, should I increase the duration of training? How many training steps do I need in each of the stages?
And I also noticed, that loss weights in your code differ from those that are written in implementation details section of the paper. Why?

**Comment by PeizhiYan** [comment]
> Hey [@PeizhiYan](https://github.com/PeizhiYan) I faced the issue with blurry results, too. If i try to train the model for larger UV map size, like 320, as you write, should I increase the duration of training? How many training steps do I need in each of the stages? And I also noticed, that loss weights in your code differ from those that are written in implementation details section of the paper. Why?

Hi!

You may increase the training steps for both stages, for example 500 steps for stage 1 and 800 steps for stage 2, to achieve better result.

Sorry for your confusion about the loss term weights. Yes the weights used in the demo code is slightly different from the details reported in the paper. But if you want to reproduce the results in our paper to compare with, you can set the weights to the same as what reported in our paper. As the weights are selected empirically, these may not be the optimal weights, you can adjust them as well.

**Comment by alasokolova** [comment]
Thanks for your answer!
I want to get as sharp and accurate results as possible, so, if the losses from the demo give better solution, i can use them. Just wondering, which ones are the best :)
And are there any other parameters I can play with to make the avatar more precise and accurate?

**Comment by PeizhiYan** [comment]
I found that the final result is very dependent on the head tracking accuracy, you may have followed our flame-head-tracker repo for the video-based FLAME tracking. But we can't promise this is the best head tracking method, if you find any other FLAME head tracking method that gives better tracking result you are free to use it as well.
Another thing is when you using larger Gaussian map (say 320x320 or even larger), it means there are more Gaussians, you may also play with the initial scale value (see ```./utils/scene/uv_gaussian.model.py``` at line #101, the initial value is -8.3533) and the scale regularization loss weight to control the size of each Gaussian within an appropriate range.
